frontend/src/components/Modal/AddPaperModal/AddPaperModal.js

```diff
@@ -23,10 +27,23 @@ class AddPaperModal extends Component {
         this.openAddPaperHandler = this.openAddPaperHandler.bind(this);
         this.checkHandler = this.checkHandler.bind(this);
         this.clickCancelButtonHandler = this.clickCancelButtonHandler.bind(this);
+        this.clickAddButtonHandler = this.clickAddButtonHandler.bind(this);
     }
 
     componentDidMount() {
+        this._isMounted = true;
 
+        this.props.onGetCollections({ id: 1 })
+            .then(() => {
+                if (this._isMounted) {
```

Comment:
Why does this code check `_isMounted`?

Reply:
It is for handling test error.

Reply:
https://www.robinwieruch.de/react-warning-cant-call-setstate-on-an-unmounted-component

Reply:
Thanks for letting me know.